examples/ezusb.h

```diff
@@ -100,7 +100,7 @@ extern int ezusb_load_eeprom(libusb_device_handle *device,
 /* Verbosity level (default 1). Can be increased or decreased with options v/q  */
 extern int verbose;
 
-extern void logerror(const char *format, ...) PRINTF_FORMAT(1, 2);
+extern void logerror(const char *format, ...);
```

Comment:
Why remove it here?

Reply:
I don't think it is actually useful to have it in this particular case. I am not 100% convinced this file should even be including config.h really. Let me run through the code one more time and think this through a bit more. I am not a fan of overloading config.h with things that we shouldn't define using m4. This macro is a good example since the different versions can be determined with a simple #if check.

Reply:
Removing this takes away the compiler's ability to warn about messing up the format string, and format string screwups can be a source of security issues.

I agree we don't need m4, autotools, cmake or anything else, we just need to create a WeWishCHadThis.h with a bunch of #if / #else to create a `PRINTF_FORMAT` macro.